protos/connectors/Events/pagerduty.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package protos.connectors.assets;
+
+import "google/protobuf/wrappers.proto";
+
+
+message PagerDutyIncidentModel {
```

Comment:
Do we need these?

Reply:
Where are we using this?

Reply:
This was for alerts later on. I'll remove them for now.